### PR TITLE
Fix bug where <figcation> is rendered as raw HTML code

### PR DESCRIPTION
**Pull request by wooseopkim**
- Before this fix:
DeepinScreenshot_select-area_20200120223808
- After this:
DeepinScreenshot_select-area_20200120223827

My Hugo version is `Hugo Static Site Generator v0.62.0/extended linux/amd64 BuildDate: unknown`. Might be related to #118.

### CODE REVIEW

File: layouts/shortcodes/figure.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {{ if .Get "src" }}
   <figure class="{{ with .Get "position"}}{{ . }}{{ else -}} left {{- end }}" >
     <img src="{{ .Get "src" | safeURL }}" {{ with .Get "alt" }} alt="{{ . | plainify }}" {{ end }} {{ with .Get "style" }} style="{{ . | safeCSS }}" {{ end }} />
-    {{ if .Get "caption" }}
+    {{- if .Get "caption" -}}
       <figcaption class="{{ with .Get "captionPosition"}}{{ . }}{{ else -}} center {{- end }}" {{ with .Get "captionStyle" }} style="{{ . | safeCSS }}" {{ end }}>{{ .Get "caption" }}</figcaption>
-    {{ end }}
+    {{- end -}}
```
Comment (panr) [inline layouts/shortcodes/figure.html]: Also undo this change.

File: layouts/shortcodes/figure.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {{ if .Get "src" }}
   <figure class="{{ with .Get "position"}}{{ . }}{{ else -}} left {{- end }}" >
     <img src="{{ .Get "src" | safeURL }}" {{ with .Get "alt" }} alt="{{ . | plainify }}" {{ end }} {{ with .Get "style" }} style="{{ . | safeCSS }}" {{ end }} />
-    {{ if .Get "caption" }}
+    {{- if .Get "caption" -}}
       <figcaption class="{{ with .Get "captionPosition"}}{{ . }}{{ else -}} center {{- end }}" {{ with .Get "captionStyle" }} style="{{ . | safeCSS }}" {{ end }}>{{ .Get "caption" }}</figcaption>
```
[the remark this answers is not in the document]
Comment (wooseopkim) [reply layouts/shortcodes/figure.html]: Acknowledged. I'll check it out, and close this if that works.

[the remark this answers is not in the document]
Comment (wooseopkim) [reply layouts/shortcodes/figure.html]: Works like a charm!

[the remark this answers is not in the document]
Comment (panr) [reply layouts/shortcodes/figure.html]: Weird... this should work. I use the pattern in footer and also Sophia made a PR for anchoring the headings using the same function... I'll look at it 🤔

[the remark this answers is not in the document]
Comment (wooseopkim) [reply layouts/shortcodes/figure.html]: As I've mentioned, I'm using `v0.62.0/extended linux/amd64 BuildDate: unknown`. Hope this helps!

**Comment by panr** [comment]
@wooseopkim TL;DR: everything seems to be working fine; I need to change the styling of caption.

--
OK, first of all, I totally misunderstood the issue, sorry... 
But I checked this, updating Hugo to `v0.63.2` and everything works fine... Don't know why you see plain HTML 🤔 

Also I think I should change styling of the caption, because (and this the funniest part)... I thought that you want to display caption as a button (filled witch green color)... So yeah, I don't understand my own code ;-P

**Comment by wooseopkim** [comment]
@panr Thanks, I'll try updating when I get home.

**Comment by panr** [comment]
I'm closing the PR due to the lack of response 😢

...but I'm gonna implement it on my own ;-)
